process/smartContract/process.go

```diff
@@ -309,6 +337,11 @@ func (sc *scProcessor) resolveBuiltInFunctions(
 		return true, err
 	}
 
+	err = sc.saveAccounts(acntSnd, acntDst)
```

Comment:
no need for this save. only at the end of the current method.

Reply:
Refactored.